x-pack/plugins/siem/public/components/timeline/body/renderers/user_host_working_dir.test.tsx

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+
+import { TestProviders } from '../../../../mock';
+
+import { UserHostWorkingDir } from '.';
+
+describe('UserHostWorkingDir', () => {
+  describe('rendering', () => {
+    test('it renders against shallow snapshot', () => {
+      const wrapper = shallow(
+        <UserHostWorkingDir
+          contextId="context-123"
+          eventId="event-123"
+          userName="[userName-123]"
+          hostName="[hostName-123]"
+          workingDirectory="[working-directory-123]"
+        />
+      );
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    test('it returns null if userName, hostName, and workingDirectory are all null', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <UserHostWorkingDir
+            contextId="context-123"
+            eventId="event-123"
+            userName={null}
+            hostName={null}
+            workingDirectory={null}
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toBeNull();
+    });
+
+    test('it returns null if userName, hostName, and workingDirectory are all undefined', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <UserHostWorkingDir
+            contextId="context-123"
+            eventId="event-123"
+            userName={undefined}
+            hostName={undefined}
+            workingDirectory={undefined}
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toBeNull();
+    });
+
+    test('it returns userName if that is all that is defined', () => {
```

Comment:
nit: it's not always clear what "all that" is referring to in some of the test descriptions

Reply:
```
if that is the only attribute defined
```

Change it to use this wording which should be clearer hopefully.